colossus-metrics/src/main/scala/colossus/metrics/StatReporter.scala

```diff
@@ -64,6 +62,19 @@ class MetricReporter(intervalAggregator : ActorRef, config: MetricReporterConfig
     }
   }
 
+  private def filterMetrics(m : MetricMap) : RawMetricMap = {
+    filters match {
+      case MetricReporterFilter.All => m.toRawMetrics
+      case MetricReporterFilter.None => Map()
```

Comment:
Will anyone actually ever use None?  Seems kinda weird if someone made a reporter and didn't report anything.